src/components/avatar/AvatarContent.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon, {types as ICON_TYPE, colors as ICON_COLOR} from '../icons/Icon';
+import {SIZE} from './avatarConstants';
+
+const AvatarContent = ({imgSrc, iconType, iconColor, iconSize}) => {
+  if (imgSrc) {
+    return <img className="sg-avatar__image" src={imgSrc}/>;
```

Comment:
I don't know if its good idea to split it that much, its same component @kdzwinel @vergilius